extension.js

```diff
@@ -91,7 +91,7 @@ function showRelated() {
   if (document) {
     currentFilename = document.fileName;
     const workspaceFolder = vscode.workspace.getWorkspaceFolder(document.uri);
-    maybeWorkspaceFolder = workspaceFolder ? workspaceFolder.uri.path : "";
+    maybeWorkspaceFolder = workspaceFolder ? removePrefixSlash (workspaceFolder.uri.path) : "";
```

Comment:
please change this to `removePrefixSlash(workspaceFolder.uri.path)` (remove the extra space)